ouroboros-consensus/src/Ouroboros/Storage/ChainDB/Impl/ImmDB.hs

```diff
@@ -195,6 +200,11 @@ defaultArgs fp = ImmDbArgs{
     , immAddHdrEnv      = error "no default for immAddHdrEnv"
     , immTracer         = nullTracer
     }
+  where
+    cacheConfig = Index.CacheConfig
+      { pastEpochsToCache = 100
+      , expireUnusedAfter = 1 * 60 -- Expire after 1 minute
```

Comment:
Change `pastEpochsToCache` to 250. Takes roughly 250 MB RAM. There are currently 166 epochs, so even one year from now, we will be able to cache all epochs in the chain. If this number were too low, i.e., less than the number of epochs that clients are requesting blocks from, we would constantly evict and reparse indices, causing a much higher CPU load.

Change `expireUnusedAfter` to 5 minutes.

Reply:
Make sure this type is exported so that it could be changed in `cardano-node` if necessary.